docs/_docs/04_reference/built-in-functions.md

```diff
@@ -685,6 +685,12 @@ carrot
   * Output contains multiple times `uuid3`, +1 more output comparing to `uuid1` and `uuid2` - because `uuid3` is declared in locals and inputs which add one more evaluation
   * Output contains only once `uuid4` since it is declared only once in `inputs`, `inputs` is not evaluated twice
 
+You can modify this caching behavior to ignore the existing directory if you know the command you are running is not dependent on the current directory path.  To do so, use the special `--terragrunt-global-cache` argument which must be passed as one of the first arguments to `run_cmd()` (and can be combined with `--terragrunt-quiet` in any order):
```

Comment:
Thanks for the PR! Could you let me know how you tested this behavior? Looking at the code above it doesn't look like they can be combined in any order.